manage/manage.js

```diff
@@ -548,7 +548,7 @@ function handleUpdate(style, {reason, method} = {}) {
     handleUpdateInstalled(entry, reason);
   }
   filterAndAppend({entry}).then(sorter.update);
-  if (!entry.matches('.hidden') && reason !== 'import') {
+  if (!entry.matches('.hidden') && reason !== 'import' && !$('#stylus-embedded-options')) {
```

Comment:
I think we can filter out these styles with `reason !== 'sync'`:
https://github.com/openstyles/stylus/blob/c50b3cfddcf566efb5b4984f95d7b21347b3baf6/background/style-manager.js#L166

Reply:
Done, thanks.